### ISSUE TITLE
ionViewDidLoad in app component doesn't get called

### ISSUE THREAD

**Issue by simonhaenisch** [ionic-team/ionic-starter-super#136]
https://github.com/ionic-team/ionic-starter-super/blob/03c246d407080eda9b3c7e2d97e4cb8091d8fe34/src/app/app.component.ts#L52-L59

This doesn't work, i. e. this doesn't get called when running a build on a mobile device. According to the [platform docs](https://ionicframework.com/docs/api/platform/Platform/#ready) `this.platform.ready()` should be called in the constructor.

**Comment by janpio** [comment]
(Could you seperate these two things into separate issues? Both are interesting, but them being in one issue will make it harder to follow up and e.g. send a link for the second to other people. Thanks.)

**Comment by janpio** [comment]
> This doesn't work, i. e. this doesn't get called when running a build on a mobile device.

Did you actually try this? And the code is never called/executed for you?

**Comment by simonhaenisch** [comment]
Yes I tried it. Just put

```ts
ionViewDidLoad() {
  console.log('app component view did load');
}
```

into your app component to see it for yourself.

By now I also tried the `source === 'cordova'` part and that seems to work (at least when running it in an emulator it shows the splash screen and status bar).

The two things are closely related because they both have to do with initializing native plugins, but sure I can open up another issue.
